README.md

```diff
@@ -12,4 +12,4 @@ For support or further information, please contact:
 
 Bhargav Kowshik
 
-Email: [bhargav\@mindshiftapps.net](mailto:bhargav@mindshiftapps.net){.email}
\ No newline at end of file
+Email: [bhargav\@Mindshiftapps.net](mailto:bhargav@Mindshiftapps.net){.email}
```

Comment:
@dogakantarci Email address is better in lowercase. Ex: `bhargav@mindshiftapps.net`

Reply:
@dogakantarci hope you will send another PR with this change.